src/data/make_dataset.py

```diff
@@ -48,38 +51,40 @@ def main():
     )  # remove duplicates
     data = data[
         data["HelpfulnessNumerator"] <= data["HelpfulnessDenominator"]
-      ]  # remove invalid data (numerator <= denominator)
+    ]  # remove invalid data (numerator <= denominator)
 
     data["Text"] = data["Text"].astype(str)
     data["Summary"] = data["Summary"].astype(str)
 
-    stop = set(stopwords.words("english"))
-    snow = nltk.stem.SnowballStemmer("english")
-    new_text_data = []
-    for sentence in data["Text"]:
-        preprocessed_sentence = preprocess_text(sentence, snow, stop)
-        new_text_data.append(preprocessed_sentence)
+
+    # stop = set(stopwords.words("english"))
+    # snow = nltk.stem.SnowballStemmer("english")
+    # new_text_data = []
+    # for sentence in data["Text"]:
+    #     preprocessed_sentence = preprocess_text(sentence, snow, stop)
+    #     new_text_data.append(preprocessed_sentence)
```

Comment:
We don't need to stem and filter stop words because of the Roberta Tokenizer?

Reply:
Yup the tokenizer already do that